ansible/roles/dashd/defaults/main.yml

```diff
@@ -8,6 +8,7 @@ dashd_zmq: false
 dashd_listen: false
 dashd_externalip: '{{ public_ip }}'
 dashd_isseednode: '{{ dashd_externalip == dashd_seednode }}'
```

Comment:
Let's use snake case for readability?

Reply:
We can, but I didn't add this code, it came from master... if you want me to fix it during this PR I can though... or we can fix it in a separate PR